app/loginManager.py

```diff
@@ -15,14 +14,22 @@ def authenticate(username, password) -> bool:
     user = UsersRepository.get_by_username(username)
 
     if user:
+        logger.debug(f"Authenticating a user, username {username} exists")
```

Comment:
what does it mean username exists? its very vague
account with username x exists and merge it with the second one so
Authenticating a user, user(or account) with username x exists, checking password